commons/src/main/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManager.java

```diff
@@ -114,11 +116,11 @@ public class UniversalRemoteStorageManager implements RemoteStorageManager {
     @Override
     public void configure(final Map<String, ?> configs) {
         Objects.requireNonNull(configs, "configs must not be null");
-        config = new UniversalRemoteStorageManagerConfig(configs);
+        final UniversalRemoteStorageManagerConfig config = new UniversalRemoteStorageManagerConfig(configs);
```

Comment:
I'm failing to see, why using the local variable over the existing `config` field?
If the existing field is not needed anymore, it should then be removed.

Reply:
Sure! removing it.